src/index.ts

```diff
@@ -1,4 +1,36 @@
-import { run } from 'probot';
+import { Server, Probot } from 'probot';
 import app from './app';
+import { register } from 'prom-client';
 
-run(app);
+async function startServer() {
+  const probotDefault = Probot.defaults({
```

Comment:
Can't we use `createProbot()` func instead?

Reply:
it is pretty much doing the same thing:
https://github.com/probot/probot/blob/7fd06d697503b45a56742aec3f4eb7e8157a7c7f/src/create-probot.ts#L36